test/ical-010.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+# ical export: encoded summary TEXT (escaped characters).
+
+. "${TEST_INIT:-./test-init.sh}"
+
+if [ "$1" = 'actual' ]; then
+  mkdir .calcurse || exit 1
+  cp "$DATA_DIR/conf" .calcurse || exit 1
+  cp "$DATA_DIR/apts-export" .calcurse/apts || exit 1
+  cp "$DATA_DIR/todo-export" .calcurse/todo || exit 1
+  "$CALCURSE" -D "$PWD/.calcurse" --export=ical | sed '/^PRODID/d'
+  rm -rf .calcurse || exit 1
+elif [ "$1" = 'expected' ]; then
+  cat <<EOD
+BEGIN:VCALENDAR
+VERSION:2.0
+BEGIN:VEVENT
+DTSTART:20200226
```

Comment:
Seems like this was generated/tested with a version of calcurse that does not include commit 0114289 (ical: use the VALUE parameter in DTSTART, 2018-05-09). I fixed this locally.